### PR TITLE
chat commands: add bird's egg sacrifices kc

### PR DESCRIPTION
**Pull request by Breina**
https://github.com/runelite/runelite/pull/18232

### CODE REVIEW

File: Chat-Commands.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ For some miscellaneous activities, the following shorthand versions can be used:
 `fishingtodt`, `fishtodt` -> `Tempoross`  
 `gotr`, `runetodt`, `rifts closed` -> `Guardians of the Rift`   
 `hunterrumour`, `hunter contract`, `hunter contracts`, `hunter tasks`, `hunter task`, `rumours`, `rumour` -> `Hunter Rumours`
```
Comment (YvesW) [inline Chat-Commands.md:184]:
```suggestion
`hunterrumour`, `hunter contract`, `hunter contracts`, `hunter tasks`, `hunter task`, `rumours`, `rumour` -> `Hunter Rumours`  
```

**Review by YvesW — COMMENTED** [review]
Could you move herbi below the hunterrumour? Then we got the following (mediocre) logic:
- First the minigames
- Hunter rumours
- Herbi (hunter, on fossil island)
- Bird egg (on fossil island)

**Comment by Breina** [comment]
@YvesW Moved. :)

Did I break your commit? I don't see the two spaces showing up.

**Comment by Felanbird** [comment]
image
they're there for me

**Pull request merged**